DetectDynamicJS.py

```diff
@@ -70,6 +70,26 @@ def doPassiveScan(self, baseRequestResponse):
         self._requestResponse = baseRequestResponse
         self._helpers = self._callbacks.getHelpers()
         
+        response = self._requestResponse.getResponse()
+        responseInfo = self._helpers.analyzeResponse(response)
+        
+        # Scan only if the statusCode is 200
+        statusCode = responseInfo.getStatusCode()
+        if statusCode != 200:
+            return None
+        
+        # Check if the script is world readable
+        resHeaders = self._helpers.analyzeResponse(response).getHeaders()
```

Comment:
I don't understand why you want it removed. You previously agreed to check statusCode, and said it's nice. Anyway, I'll remove it and commit as requested.